scripts/boxed_warm_in_docker.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+# Copyright 2018 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
```

Comment:
Can you add a comment to this script explaining the purpose (eg: `test warmer in boxed memory conditions`) and an example  of how to use it?

Reply:
I Added a documentation. I also fixed the swallowed error.